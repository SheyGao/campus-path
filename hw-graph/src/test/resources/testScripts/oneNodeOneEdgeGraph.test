# Test if a node, a edge can be added to the graph

# Create a Graph
CreateGraph graph1

# Add a node to the graph
AddNode graph1 n1

# Add a edge to the graph
AddEdge graph1 n1 n1 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1
