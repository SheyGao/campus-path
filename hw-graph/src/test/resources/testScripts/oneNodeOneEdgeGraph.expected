# Test if a node, a edge can be added to the graph

# Create a Graph
created graph graph1

# Add a node to the graph
added node n1 to graph1

# Add a edge to the graph
added edge e1 from n1 to n1 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: n1(e1)